DynamicTablesPkg/Library/Common/TableHelperLib/ConfigurationManagerObjectParser.c

```diff
@@ -949,6 +974,29 @@ PrintChar8 (
     ));
 }
 
+STATIC

```

Comment:
I just saw the PR at https://github.com/tianocore/edk2/pull/5826
and I think it is indeed a better option than what I suggested (I wanted to avoid you the trouble of making a bigger modification)

Reply:
Yeah this code was based on top of this internally, will rebase on that once that review is done.